idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/gas_phase_thermo.py

```diff
@@ -162,11 +206,34 @@ def build(self):
                              ('CO2', 3): 290, ('CO2', 4): 0,
                              ('H2O', 1): 1.7096e-8, ('H2O', 2): 1.1146,
                              ('H2O', 3): 0, ('H2O', 4): 0}
-        self.visc_d_param = Param(self.component_list,
-                                  range(1, 10),
-                                  mutable=True,
-                                  initialize=visc_d_param_dict,
-                                  doc="Dynamic viscosity constants")
+        self.visc_d_param_1 = Param(self.component_list,
+                                    mutable=True,
+                                    initialize={k: v for (k, j), v in
+                                                visc_d_param_dict.items()
+                                                if j == 1},
+                                    doc="Dynamic viscosity constants",
+                                    units=pyunits.kg/pyunits.m/pyunits.s)
```

Comment:
This unit is kg * m**-1 * s**-1 * K**-visc_d_param_2

Reply:
The equation is in this form: k=(a*(T^b))/(1+c/T +d/(T^2 ))

Reply:
Yes, this form is used for thermal conductivity and viscosity. If the units of one of those variables is [x], then the parameters should have the units {a: [x], b:[], c: [K], d: [K^2]}, correct? In the expressions the first T is divided by [K] before the exponent is applied.

Reply:
"In the expressions the first T is divided by [K] before the exponent is applied." That works, but it might get confusing for someone who compares the code to the actual equation.